From 4d65fbe3ef36e74c41d96f3c33aa3cf35ab4b09b Mon Sep 17 00:00:00 2001
From: Joseph Hall <perlhoser@gmail.com>
Date: Tue, 31 Jul 2012 05:34:27 -0600
Subject: [PATCH 12/13] Only expect args if they are actually passed in

---
 salt/modules/disk.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/salt/modules/disk.py b/salt/modules/disk.py
index 0fca708..0964962 100644
--- a/salt/modules/disk.py
+++ b/salt/modules/disk.py
@@ -33,7 +33,8 @@ def usage(args=None):
         cmd = 'df -kP'
     else:
         cmd = 'df'
-    cmd = cmd + ' -' + args
+    if args:
+        cmd = cmd + ' -' + args
     ret = {}
     out = __salt__['cmd.run'](cmd).split('\n')
     for line in out:
-- 
1.7.11.2

