diff --git a/conf/master b/conf/master
index f542051d76..fcad1961c1 100644
--- a/conf/master
+++ b/conf/master
@@ -25,7 +25,7 @@
 # permissions to allow the specified user to run the master. The exception is
 # the job cache, which must be deleted if this user is changed. If the
 # modified files cause conflicts, set verify_env to False.
-#user: root
+user: salt

 # Tell the master to also use salt-ssh when running commands against minions.
 #enable_ssh_minions: False
